intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -89,7 +89,7 @@ quantization args explanations:
 | --block_size    | block size (default: 32)                                    |
 | --scale_dtype   | fp32/bf16 type for scales (default: fp32)                   |
 | --compute_type  | Gemm computation data type: int8/fp32/ggml (default: ggml)  |
-
+| --build_dir     | build directory of binary executable files                  |
```

Comment:
can we combine model_file and build_dir ?

Reply:
model_file is path to model bin, different from build directory